### PR TITLE
Create Issue templates

### PR DESCRIPTION
**Pull request by danforbes**
- Report a Bug
- Suggest a Feature
- Ask a Question

cc @krichard410

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+title: ""
+labels: bug
+assignees: ""
+---
+
+**Description**
+
+_Tell us what happened. In particular, be specific about any changes you made to this template.
+Ideally, provide a link to your project's GitHub repository. Please note that we are not able to
+support any conceivable change to this template project, but the more information you are able to
```
Comment (JoshOrndorff) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:13]:
```suggestion
support all conceivable changes to this template project, but the more information you are able to
```

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+_What did you expect to happen after you followed the steps you described in the last section? What
+actually happened?_
+
+**Environment**
+
+_Describe the environment in which you encountered this bug. Use the list below as a starting point
+and add additional information if you think it's relevant._
+
+- Operating system:
+- Template version/tag:
```
Comment (JoshOrndorff) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:36]:
```suggestion
- Template version/tag:
- Rust version (run `rustup show`):
```

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+**Logs, Errors or Screenshots**
+
+_Please provide the text of any logs or errors that you think may be related to the bug; if
```
Comment (JoshOrndorff) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:40]:
```suggestion
_Please provide the text of any logs or errors that you experienced; if
```

**Comment by danforbes** [comment]
@JoshOrndorff - I think when you fork a repo like this GitHub may automatically turn Issues off for you, but TBH I'm not sure.

**Pull request merged**